app/views/shared/_ready_for_the_day.html.erb

```diff
@@ -0,0 +1,32 @@
+<section style="background-color:#e6e6ff;">
+ <div id="apps_section" class="container">
+   <div class="jumbotron">
+    <div class="row">
+      <h1 style="text-align:center;">Get Ready for the Day</h1>
```

Comment:
Don't use inline `style` attributes. Use a `class` on your `h1`, and style that class using css (or better: use a bootstrap class that styles it just right )